<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateUser</class>
 <widget class="QDialog" name="CreateUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Criar Usu치rio | Sistema de Controle de Comandas</string>
  </property>
  <widget class="QLabel" name="label_userCreateTitle">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>481</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Criar Usu치rio</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_userCreate">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>110</y>
     <width>491</width>
     <height>401</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushButton_userCreate">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>327</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Criar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_userCreateCancel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>327</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>21</y>
      <width>471</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_userCreate">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_userName">
       <item>
        <widget class="QLabel" name="label_userName">
         <property name="text">
          <string>Nome</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_userName">
         <property name="clearButtonEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_userEmail">
       <item>
        <widget class="QLabel" name="label_userEmail">
         <property name="text">
          <string>Email</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_userEmail"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_userPassword">
       <item>
        <widget class="QLabel" name="label_userPassword">
         <property name="text">
          <string>Senha</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_userPassword">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_userIsAdmin">
       <item>
        <widget class="QLabel" name="label_userType">
         <property name="text">
          <string>Tipo de Usu치rio:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_userType">
         <item>
          <property name="text">
           <string>Funcion치rio</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Administrador</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
